init.fish

```diff
@@ -24,12 +24,13 @@ end
 # Connect to the TMUX session if it exists, or create it if it doesn't.
 if not set -q TMUX
   set -l tmux_bin (config tmux-zen --get tmux-bin --default tmux)
+  set -l tmux_conf (config tmux-zen --get tmux-conf --default "$HOME/.tmux.conf")
   set -l session_name (config tmux-zen --get session-name --default local)
 
```

Comment:
FYI: On #15 I'm changing this command to a simple reattach or create a new session (`attach -A -s $SESSION_NAME`), which as a side effect stops creating new windows when reattaching.

(I just noticed how old this change is, this probably needs new maintainers).